enumerable.rb

```diff
@@ -122,8 +124,12 @@ def my_inject(*obj)
   end
 end
 
-def multiply_els(arr)
-  arr.my_inject(:*)
+class Multiply
+  def initialize; end
+
+  def multiply_els(arr)
+    arr.my_inject(:*)
+  end
 end
```

Comment:
- **_`#multiply_els` accepts an array as an argument and multiplies all the elements of the array together using #my_inject_**